SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift

```diff
@@ -50,6 +50,7 @@ class AnalyticsViewController : NSViewController, NSTableViewDataSource, NSTable
 
   var appCenter: AppCenterDelegate = AppCenterProvider.shared().appCenter!
 
+  @IBOutlet weak var enableManualSession: NSButton!
```

Comment:
enableManualSessionTrackerButton

Reply:
And also you need a button for starting session, like you did for other platforms.